google-cloud-tools-plugin/src/com/google/gct/login/IntelliJGoogleLoginMessageExtender.java

```diff
@@ -3,19 +3,20 @@
 import com.google.gct.idea.debugger.CloudDebugProcessStateCollector;
 import com.google.gct.idea.util.GctBundle;
 
+import com.intellij.openapi.application.ApplicationManager;
+
```

Comment:
Changes in this file will disappear when you merge master into this branch, right?